plugins/tech-radar/src/api.ts

```diff
@@ -34,11 +34,17 @@ export interface RadarQuadrant {
 export interface RadarEntry {
   key: string; // react key
   id: string;
-  moved: MovedState;
-  quadrant: RadarQuadrant;
-  ring: RadarRing;
+  quadrant: string;
   title: string;
   url: string;
+  history: Array<RadarEntryHistory>;
```

Comment:
I'm not quite getting the feeling that this conveys that `history[0]` is the current state. Wonder if something like calling it `timeline` helps at all? Another possibility would be to go with

```ts
export interface RadarEntrySnapshot {
  date: Date;
  ringId: string;
  description?: string;
  moved?: MovedState;
}

export interface RadarEntry {
  key: string;
  id: string;
  quadrant: string;
  title: string;
  url: string;
  current: RadarEntrySnapshot;
  history?: Array<RadarEntrySnapshot>;
}
```

Reply:
renamed to `timeline`, let me know if you want me to add a `current` field